packages/kbn-i18n/babel_config.js

```diff
@@ -17,18 +17,34 @@
  * under the License.
```

Comment:
__nit:__ let's rename it to `babel.config.js` as mentioned in the official docs.